lib/home/view/home_page.dart

```diff
@@ -15,7 +17,7 @@ class HomePageState extends State<HomePage> {
   final List<Widget> _screens = [
     const CollectionsScreen(),
     const RecipesScreen(viewedFromCollection: false),
-    const SyncScreen(),
+    SyncScreen(),
```

Comment:
As we discussed, we need to maintain existing import/export functionality so this isn't a breaking change. Let's keep the old `SyncScreen` as is and put this new one somewhere else